EventFilter/L1TRawToDigi/plugins/L1TRawToDigi.cc

```diff
@@ -100,14 +100,14 @@ namespace l1t {
       prov_ = PackingSetupFactory::get()->make(config.getParameter<std::string>("Setup"));
       prov_->registerProducts(*this);
 
-      slinkHeaderSize_ = config.getUntrackedParameter<int>("lenSlinkHeader", 8);
-      slinkTrailerSize_ = config.getUntrackedParameter<int>("lenSlinkTrailer", 8);
-      amcHeaderSize_ = config.getUntrackedParameter<int>("lenAMCHeader", 8);
-      amcTrailerSize_ = config.getUntrackedParameter<int>("lenAMCTrailer", 0);
-      amc13HeaderSize_ = config.getUntrackedParameter<int>("lenAMC13Header", 8);
-      amc13TrailerSize_ = config.getUntrackedParameter<int>("lenAMC13Trailer", 8);
-
-      debug_ = config.getUntrackedParameter<bool>("debug", false);
+      slinkHeaderSize_ = config.getUntrackedParameter<int>("lenSlinkHeader");
+      slinkTrailerSize_ = config.getUntrackedParameter<int>("lenSlinkTrailer");
+      amcHeaderSize_ = config.getUntrackedParameter<int>("lenAMCHeader");
+      amcTrailerSize_ = config.getUntrackedParameter<int>("lenAMCTrailer");
+      amc13HeaderSize_ = config.getUntrackedParameter<int>("lenAMC13Header");
+      amc13TrailerSize_ = config.getUntrackedParameter<int>("lenAMC13Trailer");
+
```

Comment:
This does not work with ConfDB template parsing: you must provide a value for all tracked and untracked parameters!